tools/cvm-image-rewriter/README.md

```diff
@@ -163,7 +167,7 @@ Optional
   -s <connection socket>    Default is connection URI is qemu:///system,
```

Comment:
Another thing I noticed is in the run.sh output I would slightly change the wording of the "-s" I would say "Default connection URS is qemu:///system" Is it supposed to be three ///?

Reply:
Further below you saw the word firstly.  That isn't that common of a word.  I would just use first.  That isn't super important though.

Reply:
> Another thing I noticed is in the run.sh output I would slightly change the wording of the "-s" I would say "Default connection URS is qemu:///system" Is it supposed to be three ///?

Yes, you can refer to https://libvirt.org/drvqemu.html#connections-to-qemu-driver, and according to this link it seems better to use `URI`.